velox/functions/lib/MapFromEntries.cpp

```diff
@@ -30,9 +30,11 @@ static const char* kIndeterminateKeyErrorMessage =
     "map key cannot be indeterminate";
 static const char* kErrorMessageEntryNotNull = "map entry cannot be null";
 
-// See documentation at https://prestodb.io/docs/current/functions/map.html
 class MapFromEntriesFunction : public exec::VectorFunction {
  public:
+  // @param throwOnNull If true, throws exception when input array is null or
```

Comment:
I'm not quite sure about this behavior in presto. What will happen if you run with prestissimo? will it throw exception like [here](https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/tests/MapFromEntriesTest.cpp#L92-L100)?

Reply:
Prestissimo returns NULLs.

I believe `if (inputValueVector->typeKind() == TypeKind::UNKNOWN) {` this branch is never executed because the function is marked as default-null-behavior, hence, the expression evaluation never invokes this function on NULL inputs. I guess we should just remove this code.